libraries/gpu/src/gpu/GLBackendTransform.cpp

```diff
@@ -224,11 +218,14 @@ void GLBackend::updateTransform(const Batch& batch) {
             glEnableVertexAttribArray(gpu::Stream::DRAW_CALL_INFO); // Make sure attrib array is enabled
             glVertexAttribIFormat(gpu::Stream::DRAW_CALL_INFO, 2, GL_UNSIGNED_SHORT, 0);
             glVertexAttribBinding(gpu::Stream::DRAW_CALL_INFO, gpu::Stream::DRAW_CALL_INFO);
-            glVertexBindingDivisor(gpu::Stream::DRAW_CALL_INFO, 1);
             _transform._enabledDrawcallInfoBuffer = true;
         }
-        glBindVertexBuffer(gpu::Stream::DRAW_CALL_INFO, _transform._drawCallInfoBuffer, (GLintptr)_transform._drawCallInfoOffsets[batch._currentNamedCall], 0);
-
+        // NOTE: A stride of zero in BindVertexBuffer signifies that all elements are sourced from the same location,
```

Comment:
See the issues section of [the spec](https://www.opengl.org/registry/specs/ARB/vertex_attrib_binding.txt).